pyro/optim/lr_scheduler.py

```diff
@@ -18,6 +18,7 @@ class PyroLRScheduler(PyroOptim):
         pyro_scheduler = pyro.optim.ExponentialLR({'optimizer': optimizer, 'optim_args': {'lr': 0.01}, 'gamma': 0.1})
         svi = SVI(model, guide, pyro_scheduler, loss=TraceGraph_ELBO())
         svi.step()
+        pyro_scheduler.step(epoch=epoch)
```

Comment:
can we please update this to be a full and complete snippet with comments, something like
```
for epoch in range(num_epochs):
    blah() # this happens once an epoch
    for minibatch in loader:
        svi.step() # this takes single gradient step and generally happens many times per epoch
```

Reply:
also please double-check the docstring for correctness/completeness in light of changes